src/aws-util/Cargo.toml

```diff
@@ -15,6 +15,9 @@ aws-sdk-s3 = { version = "0.5.0", default-features = false, features = ["native-
 aws-sdk-sqs = { version = "0.5.0", default-features = false, features = ["native-tls"], optional = true }
 aws-sdk-sts = { version = "0.5.0", default-features = false, features = ["native-tls"], optional = true }
 aws-types = { version = "0.5.0" }
+# Make sure the "vendored" feature makes it into the transitive dep graph of
+# every aws user, so the right things are available at link time.
```

Comment:
I'd probably say something like "so that we don't attempt to link against the system OpenSSL library" but I'm not fussed about it.